res/values/strings.xml

```diff
@@ -802,6 +802,8 @@
     <!-- %1$s will be replaced by name and address of the contact who did the action -->
     <string name="group_image_changed_by_other">Group image changed by %1$s.</string>
     <!-- %1$s will be replaced by name and address of the contact added to the group -->
+    <string name="member_added">Member %1$s added.</string>
```

Comment:
to not confuse with the plain text "member added" (even if not in used currently), maybe change the key to:
```suggestion
    <string name="member_x_added">Member %1$s added.</string>
```